src/web/app/pages-instructor/instructor-course-edit-page/instructor-edit-panel/instructor-edit-panel.component.ts

```diff
@@ -8,7 +8,7 @@ import {
  * Model for edit instructor panel.
  */
 export interface InstructorEditPanel {
-  googleId: string;
+  googleId?: string;
```

Comment:
https://github.com/TEAMMATES/teammates/blob/1df7a07640bb533ab4202a2b745107e2e64cfd5e/src/web/app/pages-instructor/instructor-course-edit-page/instructor-edit-panel/instructor-edit-panel.component.html#L40-L50

Need to modify the front-end slightly. This row should be visible only if the user has `canmodifyinstructor` permission.